Makefile

```diff
@@ -166,6 +173,27 @@ wait-kubernetes-dependencies: ## Wait for dependencies to be ready
 	  --selector=app.kubernetes.io/component=controller \
 	  --timeout=90s
 
+update-version: ## update release version
+	find . -type f | xargs sed -i 's/$(VERSION)/${RELEASE_VERSION}/g'
+
+bump-version: ## update post-release version and update commit message
+	find . -type f | xargs sed -i 's/$(RELEASE_VERSION)/${NEXT_VERSION}/g'
+
+git-commit: ## adds all files and bumps the version
```

Comment:
This is too general target name, need to have name that clarify it only for version bump
suggestion: `bump-git-commit`
Or maybe this target can be merged with `bump-version` target?

Reply:
Will change the name. Merging both targets is not good idea because they do not run one after the other.